### PR TITLE
Adding support for "Accept: application/json" to route GET /scheduler…

### PR DESCRIPTION
**Pull request by matheuscscp**
…/{schedulerName}/config

### CODE REVIEW

File: api/scheduler_config_handler.go [hunk 1]
```diff
@@ -6,6 +6,7 @@ import (
 
 	"github.com/Sirupsen/logrus"
 	"github.com/topfreegames/maestro/models"
+	ghodssYaml "github.com/ghodss/yaml"
```
Comment (Henrod) [inline api/scheduler_config_handler.go]: This import is not necessary. You can convert to json using the encoding/json package.

File: api/scheduler_config_handler.go [hunk 2]
```diff
@@ -47,6 +48,20 @@ func (g *GetSchedulerConfigHandler) ServeHTTP(w http.ResponseWriter, r *http.Req
 		return
 	}
 
+	// json support
+	acceptHeader := r.Header.Get("Accept")
+	if acceptHeader == "application/json" {
```
Comment (Henrod) [inline api/scheduler_config_handler.go]: This is not necessary and can add some headache.

File: api/scheduler_config_handler_test.go [hunk 3]
```diff
@@ -67,13 +68,29 @@ var _ = Describe("SchedulerConfigHandler", func() {
...
+		})
+
 		It("should return config yaml of specified version", func() {
 			version := "v1.0"
 			MockSelectYamlWithVersion(yamlString, version, mockDb, nil)
 
 			url = fmt.Sprintf("http://%s/scheduler/%s/config?version=%s",
 				app.Address, configYaml.Name, version)
 			request, _ = http.NewRequest("GET", url, nil)
+			request.SetBasicAuth(user, pass)
```
Comment (Henrod) [inline api/scheduler_config_handler_test.go]: By removing on BeforeEach and adding this, you are removing the test with Oauth. 
Please, add a new test that uses oauth.

File: api/scheduler_config_handler_test.go [hunk 4]
```diff
@@ -40,17 +42,16 @@ var _ = Describe("SchedulerConfigHandler", func() {
 			Do(func(destToken *login.DestinationToken, query string, modifier string) {
 				destToken.RefreshToken = "refresh-token"
 			}).AnyTimes()
-		mockLogin.EXPECT().
```
Comment (Henrod) [inline api/scheduler_config_handler_test.go]: It is important to test with oauth too.

File: api/scheduler_config_handler.go (hunk 1 above)
Comment (matheuscscp) [reply api/scheduler_config_handler.go]: Even if the input byte stream is a YAML text?

Comment (Henrod) [reply api/scheduler_config_handler.go]: yes

**Pull request merged**